assets/js/components/performance/Table/scripts.jsx

```diff
@@ -308,6 +294,19 @@ class TabularView extends Component {
         }
     }
 
+    handleObservers() {
+        const ps = document.querySelectorAll('.performance-table-cell span');
+        const observer = new ResizeObserver(entries => {
+            for (let entry of entries) {
+                entry.target.classList[entry.target.scrollHeight * 0.95 > entry.contentRect.height ? 'add' : 'remove']('truncated');
+            }
+        })
+
+        ps.forEach(p => {
+            observer.observe(p);
+        })
+    }
+
```

Comment:
is this a resource leak? how do we clean them up?
Looks like memory consumption is going up every time I click forward or back
![Uploading Screenshot_2023-03-14_12-55-17.png…]()

Reply:
It looks like in prod it's garbage collecting. It might be because it's a prod js bundle - could you just look at garbage collection for the observers we're creating, and whether memory doesn't just grow infinitely as you use the UI?

![Screenshot_2023-03-14_13-05-40](https://user-images.githubusercontent.com/235801/224982534-311a7889-ffa3-40b5-ad95-864419a61581.png)

![Screenshot_2023-03-14_12-55-17](https://user-images.githubusercontent.com/235801/224982543-a0879f15-9808-42f1-9611-d41b98468713.png)